demo/src/main/java/com/bluelinelabs/conductor/demo/changehandler/CityGridSharedElementTransitionChangeHandler.java

```diff
@@ -5,18 +5,19 @@
 import android.os.Bundle;
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
-import android.transition.ChangeBounds;
-import android.transition.ChangeClipBounds;
-import android.transition.ChangeTransform;
-import android.transition.Explode;
-import android.transition.Slide;
-import android.transition.Transition;
-import android.transition.TransitionSet;
+import androidx.transition.ChangeBounds;
+import androidx.transition.ChangeClipBounds;
+import androidx.transition.ChangeTransform;
+import androidx.transition.Explode;
+import androidx.transition.Slide;
+import androidx.transition.Transition;
+import androidx.transition.TransitionSet;
+
 import android.view.Gravity;
 import android.view.View;
 import android.view.ViewGroup;
 
-import com.bluelinelabs.conductor.changehandler.SharedElementTransitionChangeHandler;
+import com.bluelinelabs.conductor.changehandler.androidxtransition.SharedElementTransitionChangeHandler;
```

Comment:
`androidxtransition` package naming is so bad. Maybe just `support.changehandler.transition`?

Reply:
androidx.transition

Reply:
Mb, but my point is that AndroidX is the part of support, and we already have `support` package, so there is no AndroidX mentioning in the package names.

Reply:
But honestly everything is better than `androidxtransition`.

Reply:
For me the `support.changehandler` is the Best option

Reply:
ok, I see